### ISSUE TITLE
Need example of how to use: write-object-property

### ISSUE THREAD

**Issue by ronniebach** [asad-boutique/node-bacnet-contrib-extended#8]
The documentation gives no specification on what parameter/objects to send to write-object-property ?

Could someone give an example of how it works?

Thanks.

**Comment by ReachDS** [comment]
Agreed, this would be helpful. Currently it seems the write-object-property is still a WIP. Certainly, this node will include additional parameters such as a BACnet Priority number for the object write. Looking forward to a working version of this node. All of the other odes have worked great.
Thanks

**Comment by NickKh00** [comment]
Heys so after scrubbing though the code, I've gotten the write to function properly. 
Here's the test flow:
`[
    {
        "id": "4407299b0cac76a7",
        "type": "write-object-property",
        "z": "e59ccc493ab8713f",
        "name": "",
        "x": 800,
        "y": 760,
        "wires": [
            [
                "37598b91a7510ae0"
            ]
        ]
    },
    {
        "id": "e6347c9deb0f9ce5",
        "type": "function",
        "z": "e59ccc493ab8713f",
        "name": "Inject Id + Type + Properties",
        "func": "msg.deviceId = 300;\nmsg.address = \"192.168.20.250\";\nmsg.propertyId =85;\nmsg.objectId = { type: 5, instance: 10 };\nmsg.values = [{ type: 2, value: 0 }];\n\nreturn msg;\n//refer to https://fh1ch.github.io/node-bacstack/bacstack.html#.writeProperty",
        "outputs": 1,
        "noerr": 0,
        "initialize": "",
        "finalize": "",
        "libs": [],
        "x": 500,
        "y": 760,
        "wires": [
            [
                "4407299b0cac76a7"
            ]
        ]
    },
    {
        "id": "beede5c456eb2368",
        "type": "inject",
        "z": "e59ccc493ab8713f",
        "name": "",
        "props": [
            {
                "p": "payload"
            },
            {
                "p": "topic",
                "vt": "str"
            }
        ],
        "repeat": "",
        "crontab": "",
        "once": false,
        "onceDelay": 0.1,
        "topic": "",
        "payload": "",
        "payloadType": "date",
        "x": 220,
        "y": 760,
        "wires": [
            [
                "e6347c9deb0f9ce5"
            ]
        ]
    },
    {
        "id": "37598b91a7510ae0",
        "type": "debug",
        "z": "e59ccc493ab8713f",
        "name": "debug 29",
        "active": true,
        "tosidebar": true,
        "console": false,
        "tostatus": false,
        "complete": "false",
        "statusVal": "",
        "statusType": "auto",
        "x": 1000,
        "y": 760,
        "wires": []
    }
]`

**Comment by ReachDS** [comment]
Thanks for your efforts. Will try the new node accordingly. 
I still see that sending a "Priority" number is missing and the logical "Relinquish" control for a BACnet output.
Is it functionable?
Thanks again.

**Comment by NickKh00** [comment]
Yeah with the system I'm testing with it works exactly as expected. 

If we look at line 54 in write-object-property.js
It's written in the same manner as the example here: https://fh1ch.github.io/node-bacstack/bacstack.html#.writeProperty
Completely ignoring the option arguments, I assume for dev since they're labeled as optional.

**Comment by NickKh00** [comment]
I checked, by not specifying the priority, the function writes to priority 16

**Comment by optimalcontrolssac-crypto** [comment]
I used your example but only gives me: "Error: BacnetError Class: PROPERTY (2) Code: INVALID_DATA_TYPE (9)"
